internal/fvalidate/verapdf_validator.go

```diff
@@ -0,0 +1,76 @@
+package fvalidate
+
+import (
+	"errors"
+	"os/exec"
+
+	"github.com/go-logr/logr"
+)
+
+// pdfaPUIDs are the https://www.nationalarchives.gov.uk/pronom/ IDs of the
+// PDF/A formats.
+var pdfaPUIDs = []string{
+	"fmt/95",   // PDF/A 1a
+	"fmt/354",  // PDF/A 1b
+	"fmt/476",  // PDF/A 2a
+	"fmt/477",  // PDF/A 2b
+	"fmt/478",  // PDF/A 2u
+	"fmt/479",  // PDF/A 3a
+	"fmt/480",  // PDF/A 3b
+	"fmt/481",  // PDF/A 3u
+	"fmt/1910", // PDF/A 4
+	"fmt/1911", // PDF/A 4e
+	"fmt/1912", // PDF/A 4f
+}
+
+type veraPDFValidator struct {
+	cmd    string
+	logger logr.Logger
+}
+
+var _ Validator = (*veraPDFValidator)(nil)
+
+func NewVeraPDFValidator(cmd string, logger logr.Logger) *veraPDFValidator {
+	return &veraPDFValidator{cmd: cmd, logger: logger}
+}
+
+func (v *veraPDFValidator) FormatIDs() []string {
+	return pdfaPUIDs
+}
+
+func (v *veraPDFValidator) Name() string {
+	return "veraPDF"
+}
+
+func (v *veraPDFValidator) Validate(path string) (string, error) {
+	// If the veraPDF cmd path is not set then skip validation.
+	if v.cmd == "" {
+		return "", nil
+	}
+
+	cmd := exec.Command(v.cmd, "--recurse", path) // #nosec: G204 -- trusted path.
```

Comment:
We may want to add allow configuring additional args (like "--flavour" or "--defaultFlavour") in `fvalidate.Config`, but it might also be an additional security risk.  I'm not sure if there's a reasonable way to sanitize `cmd` or if it's reasonable to just trust that the config isn't malicious.

Reply:
To be clear I was wondering about adding a way to specify general arguments for verapdf in the config file, something like:

```
args := append(cfg.args, path) 
cmd := exec.Command(v.cmd, "--recurse", args...)
```

Where `cfg.args` would be a slice of string arguments.

I'm not sure how adding a way to add arguments in config would add a version checking requirement?